rocketpy/plots/plot_helpers.py

```diff
@@ -0,0 +1,70 @@
+from pathlib import Path
+import warnings
+
+from matplotlib.figure import Figure
+import matplotlib.pyplot as plt
+
+SAVEFIG_DPI = 300
+
+
+def show_or_save_plot(filename=None):
+    """Shows or saves the current matplotlib plot. If a filename is given, the plot will be saved, otherwise it will be shown.
+
+    Parameters
+    ----------
+    filename : str | None, optional
+        The path the plot should be saved to, by default None. Supported file endings are: eps, jpg, jpeg, pdf, pgf, png, ps, raw, rgba, svg, svgz, tif, tiff and webp.
+    """
+    if filename == None:
+        plt.show()
+    else:
+        # Warn if file ending is not supported
+        file_ending = Path(filename).suffix
+        if file_ending not in get_matplotlib_supported_file_endings():
+            warnings.warn(
+                "Warning: Unsupported file ending '" + file_ending + "'!", UserWarning
+            )
```

Comment:
Python f-strings are usually faster, and also easier to read and write.

```suggestion
            warnings.warn(
                f"Warning: Unsupported file ending '{file_ending}'!", UserWarning
            )

```

What do you think> This is just a suggestion tho, feel free to not apply it if you don't like it.

Reply:
Good point, I will apply this